unstructured/utils.py

```diff
@@ -26,13 +28,18 @@
 )
 
 import requests
-from typing_extensions import ParamSpec
+from typing_extensions import ParamSpec, TypeAlias
 
 from unstructured.__version__ import __version__
 
 if TYPE_CHECKING:
     from unstructured.documents.elements import Text
 
+Point: TypeAlias = Tuple[float, float]
+Points: TypeAlias = Tuple[Point, ...]
+# Box format: [x_bottom_left, y_bottom_left, x_top_right, y_top_right]
+Box: TypeAlias = Tuple[float, float, float, float]
```

Comment:
Generally a good idea to organize types like this in alphabetical order to ease scanning.